modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -160,7 +179,7 @@ function getOSMTileIndices(viewport, z) {
  * than minZoom, return an empty array. If the current zoom level is greater than maxZoom,
  * return tiles that are on maxZoom.
  */
-export function getTileIndices(viewport, maxZoom, minZoom, tileSize = TILE_SIZE) {
+export function getTileIndices(viewport, maxZoom, minZoom, zRange, tileSize = TILE_SIZE) {
```

Comment:
The failing test is because I put `zRange` before `tileSize`. So when
```js
getTileIndices(
      testCase.viewport,
      testCase.maxZoom,
      testCase.minZoom,
      testCase.tileSize
    );
```
is called, `tileSize` is passed as the `zRange` positional argument. How should I deal with this? Should I make `zRange` the last positional argument? Or should I change the test cases to add a `zRange`?

Reply:
You can change the test cases. This is not a public API.